### PR TITLE
adding a club entity

### PR DESCRIPTION
**Pull request by olfa-koubaa**

### CODE REVIEW

File: src/main/java/insat/company/platform/domain/Club.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "president")
+    private String president;
```
Comment (SkanderMansouri) [inline src/main/java/insat/company/platform/domain/Club.java:28]: This should be a relationship with a User not a String

File: src/main/java/insat/company/platform/domain/Club.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    @Column(name = "field")
+    private String field;
```
Comment (SkanderMansouri) [inline src/main/java/insat/company/platform/domain/Club.java:31]: What do you mean by field?

File: src/main/java/insat/company/platform/service/dto/ClubDTO.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
+package insat.company.platform.service.dto;
```
Comment (SkanderMansouri) [inline src/main/java/insat/company/platform/service/dto/ClubDTO.java:1]: we don't need a DTO for club

File: src/main/java/insat/company/platform/service/mapper/ClubMapper.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package insat.company.platform.service.mapper;
```
Comment (SkanderMansouri) [inline src/main/java/insat/company/platform/service/mapper/ClubMapper.java:1]: we don't need a club mapper

File: src/main/java/insat/company/platform/service/mapper/EntityMapper.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
+package insat.company.platform.service.mapper;
```
Comment (SkanderMansouri) [inline src/main/java/insat/company/platform/service/mapper/EntityMapper.java:1]: No need for this too

File: src/main/java/insat/company/platform/domain/Club.java (hunk 1 above)
Comment (olfa-koubaa) [reply src/main/java/insat/company/platform/domain/Club.java:28]: ok I'll fix it

File: src/main/java/insat/company/platform/domain/Club.java (hunk 2 above)
Comment (olfa-koubaa) [reply src/main/java/insat/company/platform/domain/Club.java:31]: domaine : it / security ....

File: src/main/java/insat/company/platform/service/dto/ClubDTO.java (hunk 3 above)
Comment (olfa-koubaa) [reply src/main/java/insat/company/platform/service/dto/ClubDTO.java:1]: ok

File: src/main/java/insat/company/platform/service/mapper/ClubMapper.java (hunk 4 above)
Comment (olfa-koubaa) [reply src/main/java/insat/company/platform/service/mapper/ClubMapper.java:1]: ok

File: src/main/java/insat/company/platform/service/mapper/EntityMapper.java (hunk 5 above)
Comment (olfa-koubaa) [reply src/main/java/insat/company/platform/service/mapper/EntityMapper.java:1]: ok

File: src/main/java/insat/company/platform/domain/Club.java (hunk 2 above)
Comment (SkanderMansouri) [reply src/main/java/insat/company/platform/domain/Club.java:31]: It should be an Enum then, create an Enum called Field and have these static value there.

**Pull request closed without merging**